frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/project/useProjectSearch.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useQuery } from "react-query";
+import { WorksService } from "../../services/elements/Works";
+
+//using this hook for searching only one project with projectNumber/id etc
+const useProjectSearch = ({ tenantId, searchParams, filters = { limit: 10, offset: 0, includeAncestors: true, includeDescendants: true }, config = {} }) => useQuery(
```

Comment:
Can we use 'useViewProjectDetails' hook for this ?

Reply:
using it for some basic details, no need to search for parent/child projects. I have removed the includeAncestors and descendants flags